deploy/workloads/memcached-thread-imbalance-config.yaml

```diff
@@ -1,42 +1,42 @@
-run-farm:
-    run-farm-tag: memcached-mainrunfarm
-    f1_16xlarges: 3
-    m4_16xlarges: 0
+run_farm:
+    run_farm_tag: memcached-mainrunfarm
+    f1-16xlarges: 3
```

Comment:
Why are these hyphenated now?

Reply:
I tried to just modify the keys of the API, not the values (only the left-side). If we want to change all of these as well, I can do this also.

Reply:
I'm talking about the `f1-16xlarge` on the LHS.

Reply:
Oh. That's something I missed. In general, I haven't tested this yet (since I was waiting on general feedback). For now, just assume that if you see something that looks like an error (it probably is). Thanks for pointing it out though.